Lib/test/test_sysconfig.py

```diff
@@ -272,8 +273,19 @@ def test_user_similar(self):
                 # before comparing
                 global_path = global_path.replace(sys.base_prefix, sys.prefix)
                 base = base.replace(sys.base_prefix, sys.prefix)
+
+            if platsubdir != "lib":
```

Comment:
I don't like how we're special casing "lib" here. I'd prefer to always take the same code path and use some special token that gets replaced.

Reply:
this is the same thing the code did before though.
i suppose we could introduce `puresubdir` as the other option and prefill it to `lib` somewhere, is that what you're suggesting?

Reply:
hm, no, i misread which part of the code you're commenting. sorry.